sdk/core/Azure.Core/tests/FailedResponseExceptionTests.cs

```diff
@@ -312,21 +315,24 @@ public void ParsesJsonErrors()
         }
 
         [Test]
-        public void ParsesJsonErrors_ResponseCtor()
+        public void ParsesJsonErrors_ResponseCtor([Values(true, false)] bool hasErrorWrapper)
```

Comment:
Any reason to use Values vs the more intuitive TestCase?

Reply:
Not a big difference in this case, but it's a much more concise way to do permutations of values.

Reply:
I see - if there are multiple parameters you can avoid repeating ones that shouldn't change. Being that this is not the case here, could we use TestCase instead?

Reply:
We were already using `Values` for `canseek` and the permutations behavior is helpful there, so I prefer to be consistent and leave it as is, unless there is a more compelling reason to change.